docs/level-23.md

```diff
@@ -33,6 +33,7 @@ import TrashPush from '@site/image-generator/yml/level-23/trash-push.yml';
 ### Unnecessary Moves with Known Trash --> Ignition
 
 - If **known trash** is used as clue target, the "extra" thing is an *Ignition*. The *Ignition* can be on anyone on the team, similar to how *Double Ignition* works.
+  	- Applicable conventions include *Trash Push*, *Trash Push Finesse*, *Trash Pull*, and performing a *Trash Chop Move* on a card which can be clued directly.
```

Comment:
i'd rather not include explicit enumeration of moves here since
1) it strikes me as being unnecessary (no pun intended)
2) it has the potential to get out of date when we add new moves